packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -45,10 +45,23 @@
 	margin: $grid-unit-20;
 
 	.component-color-indicator {
-		// Show a diagonal line (crossed out) for empty swatches.
-		background: linear-gradient(-45deg, transparent 48%, $gray-300 48%, $gray-300 52%, transparent 52%);
```

Comment:
What if you did this:

`background: $white linear-gradient(-45deg, transparent 48%, #ddd 48%, #ddd 52%, transparent 52%)`

?

Then perhaps you wouldn't need a pseudo element 🤔